src/assets/locales/resources_en_US.json

```diff
@@ -31,6 +31,30 @@
     "_description": "Label for the repeater item checkbox for the repeater component",
     "message": "Select row"
   },
+  "skyux_repeater_item_reorder_cancel": {
+    "_description": "Accessible screen reader text for when a reorder is finished",
+    "message": "Item rewording was canceled. The item was returned to its original position."
```

Comment:
`rewording` should be `reordering`. This was a typo on my part when I provided feedback yesterday. Sorry about that.

Reply:
Done